src/routes/trade/trade.jsx

```diff
@@ -170,16 +174,19 @@ const Trade = () => {
             <section
               class={classNames(
                 styles.container,
-                dashboardStyles["market-value"]
+                styles[`container--${status()}`]
+                // dashboardStyles["market-value"],
+                // dashboardStyles[`market-value--${status()}`]
```

Comment:
commented lines. corresponding styles from the scss file as well to be removed.

Reply:
We are using these styles in different components.I will remove these commented lines.Thanks